src/services/modal.service.js

```diff
@@ -0,0 +1,4 @@
+export default function getModalName(classNameString){
```

Comment:
This function is specific for optimizely sign up modal. So `getModalName` is to generic for this. Did you even check how other *.service.js files look? Put this function in `src/service/ui.service.js` file and name it `getOptimizelyModalName`.

Also `className` is enough as argument name.